docs/other-inverters.md

```diff
@@ -65,7 +65,8 @@ Discussion ticket is here: <https://github.com/springfall2008/batpred/issues/181
 
 - Please copy the template apps.yaml from <https://github.com/springfall2008/batpred/blob/main/templates/solaredge.yaml> and modify for your system
 - Ensure that **number.solaredge_i1_storage_command_timeout** is set to reasonably high value e.g. 3600 seconds to avoid the commands issued being cancelled
-- Power Control Options, as well as Enable Battery Control, must be enabled in the Solaredge Modbus Multi integration configuration, and switch.solaredge_i1_advanced_power_control must be on.
+- Power Control Options, as well as Enable Battery Control, must be enabled in the Solaredge Modbus Multi integration configuration,
+and switch.solaredge_i1_advanced_power_control must be on.
```

Comment:
isn't this invalid markdown?

Reply:
What's wrong with the line, sorry not sure?